integration/fuse/src/test/java/alluxio/fuse/ufs/stream/InOrOutStreamOutTest.java

```diff
@@ -51,6 +51,10 @@ protected FuseFileStream createStream(AlluxioURI uri, boolean truncate) {
   @Test(expected = UnimplementedRuntimeException.class)
   public void read() throws Exception {
     AlluxioURI alluxioURI = getTestFileUri();
+    if (alluxioURI.getParent().getPath().length() > 1 ) {
```

Comment:
```suggestion
    if (!alluxioURI.getParent().isRoot()) {
```

Reply:
Thank you.
Sorry for confusion, but this change is removed.